pkg/operator/configobservation/network/observe_network.go

```diff
@@ -14,39 +14,32 @@ import (
 
 // ObserveRestrictedCIDRs watches the network configuration and updates the
 // RestrictedEndpointsAdmission controller config.
-func ObserveRestrictedCIDRs(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
-	listers := genericListers.(configobservation.Listers)
+func ObserveRestrictedCIDRs(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (ret map[string]interface{}, _ []error) {
```

Comment:
@squeed @rphillips please review

Reply:
Wait, should we be returning the pruned config (i.e. only keys we care about), or the full Existing on failure?

Reply:
only pruned. We forgot that everywhere. Hence, if some observer errored, other observers potentially stopped working (their config was overwritten by the one with the error).